tap_google_ads/streams.py

```diff
@@ -230,13 +230,14 @@ def on_giveup_func(err):
 @backoff.on_exception(backoff.expo,
                       (GoogleAdsException,
                        ServerError, TooManyRequests,
-                       ReadTimeout,
+                        ReadTimeout,
                        AttributeError),
                       max_tries=5,
                       jitter=None,
                       giveup=should_give_up,
                       on_giveup=on_giveup_func,
                       logger=None)
+@backoff.on_exception(backoff.expo, Unauthorized, max_tries=5, jitter=None,logger=None)
```

Comment:
Can't we append Unauthorized in 1st backoff itself?

Reply:
the above exceptions have a on_giveup handler, which is not needed and does not work with this specific exception class hence added a new decorator